plugins/json/core/src/test/java/json/JsonSamplerTest.java

```diff
@@ -20,13 +20,18 @@
  *
  ******************************************************************************/
 
-package org.pentaho.getfields.types.json;
+package json;
```

Comment:
@hv-vamsi Please move this to the appropriate package... org.pentaho.di.trans.steps.<xxx> where <xxx> is meaningful.